client/components/LearnerViewDirective.js

```diff
@@ -23,12 +23,19 @@ tie.directive('learnerView', [function() {
     template: `
       <div class="tie-exercise-ui-outer">
         <div class="tie-exercise-ui-inner">
-          <div>
-            Stepper:
-            <div ng-repeat="questionId in questionIds track by $index">
-              <span ng-class="{'tie-active-question-index': currentQuestionIndex === $index}">
-                {{$index + 1}}
-              </span>
+          <div class="tie-step-container-outer">
+            <div class="tie-step-container-inner">
+              <div class="tie-step-item" ng-repeat="questionId in questionIds\
```

Comment:
Don't end lines with backslash please -- hard to tell what char is being escaped, there could be a space at the end. Prefer long lines to ending with backslash, or alternatively you could move ng-repeat to next line.